python/singa/engine.py

```diff
@@ -0,0 +1,95 @@
+from collections import Counter
+from singa import singa_wrap as singa
+
+
+class ExecutionEngine(object):
```

Comment:
need to add comments for class and functions.
like https://github.com/apache/incubator-singa/blob/master/python/singa/tensor.py#L73 and https://github.com/apache/incubator-singa/blob/master/python/singa/tensor.py#L103